core-web/libs/sdk/analytics/src/lib/react/internal/uve.utils.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * Checks if the code is running inside the DotCMS Universal Visual Editor (UVE).
+ *
+ * @returns {boolean} Returns true if running inside the editor, false otherwise.
+ *
+ * @example
+ * ```ts
+ * if (isInsideUVE()) {
+ *   // Running inside UVE editor
+ *   console.log('Inside editor');
+ * } else {
+ *   // Running in normal mode
+ *   console.log('Outside editor');
+ * }
+ * ```
+ */
+export function isInsideUVE(): boolean {
+    if (typeof window === 'undefined' || window.parent === window || !window.location) {
+        return false;
+    }
```

Comment:
This might be weak beacuse it's checking if the page is not inside an iframe, not specifically in the UVE. Can you use `getUVEState()`?  That's also weak but it's a bit more robust